### PR TITLE
HL-142

### PR DESCRIPTION
**Pull request by brettvniekerk**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82)

- Fixes race status interaction
- Fixes payout

### CODE REVIEW

**Review by OptimusOpus — APPROVED** [review]

File: src/components/Dashboard/DashboardTable.tsx [hunk 1]
```diff
@@ -59,14 +59,18 @@ export const DashboardTable: React.FC<Props> = ({ meets }) => {
...
+                                    : race.status == "Interim"
+                                    ? "bg-emerald-400"
                                     : "hover:bg-gray-200"
                                 }`}
                             >
                               {race.name ? (
                                 <Link
                                   to={
-                                    race.status !== "Paying"
+                                    race.status == "Normal"
```
Comment (bitcoinbrisbane) [inline src/components/Dashboard/DashboardTable.tsx:70]: Should this be...

```suggestion
                                    race.status === "Normal"
```

Comment (brettvniekerk) [reply src/components/Dashboard/DashboardTable.tsx]: done

File: src/components/Dashboard/DashboardTable.tsx [hunk 2]
```diff
@@ -59,14 +59,18 @@ export const DashboardTable: React.FC<Props> = ({ meets }) => {
...
                                       ? `/horses/${meet.id}/${race.number}`
+                                      : race.status == "Interim"
```
[the remark this answers is not in the document]
Comment (brettvniekerk) [reply src/components/Dashboard/DashboardTable.tsx]: done

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by JasminFarhi — APPROVED** [review]
LGTM

**Pull request merged**